README.md

```diff
@@ -69,8 +69,8 @@ The creation of the chameleon sprite can be customized with various options. All
 ```javascript
 module.exports = {
   path: 'path/to/svg/directory/',           // default: '' (current working directory)
-  subdirName: 'my-sprite-dir',              // default: 'chameleon-sprite' (created inside your SVG directory, stores all generated files)
-  fileName: 'my-sprite',                    // default: 'chameleon-sprite' (used for .svg, .scss and .css files)
+  dest: 'path/to/dest/directory/'           // default: path + name as subfolder, if dest is specified no additional subfolder is created
+  name: 'my-sprite',                        // default: 'chameleon-sprite' (used for .svg, .scss and .css files)
   scss: true,                               // default: false (creates scss with classes for dimensions)
   css: true,                                // default: false (creates css with classes for dimensions)
```

Comment:
@harterSe dest on main object looks good, but the restructuring of the scss and css options is still todo! See my comment on the issue:

```
dimensionStyles: {
    css: {
      create: false, // automatically true if other option is set
      dest: null, // default same dir as sprite
      name: null // default is name
    },
    scss: {
      create: false, // automatically true if other option is set
      dest: null, // default same dir as sprite
      name: null // default is name
    },
  },
```
to keep the cli options simple i would leave --scss and --css to set dimensionStyles.scss/css.create  and expand it with --scss-dest, --scss-name, --css-dest, --css-name